### ISSUE TITLE
A description problem in tf.keras.layers.MaxPooling2D

### ISSUE THREAD

**Issue by ShenghanZhang** [tensorflow/tensorflow#44216]
## URL(s) with the issue:
https://keras.io/api/layers/pooling_layers/max_pooling2d/

## Description of issue (what needs changing):

At the end of second paragraph, you give a fomula to compute the size of output without padding: 'output_shape = (input_shape - pool_size + 1) / strides)'. However, there is something wrong with it. The correct one should be 'output_shape = (input_shape - pool_size) / strides + 1'. 
Thank you!

**Comment by Molkree** [comment]
I believe this was fixed in 31969210c748b85e8f7ff5c0dd64ced988bdb5f6, but this change didn't make it into [2.3](https://github.com/tensorflow/tensorflow/blob/r2.3/tensorflow/python/keras/layers/pooling.py#L337) branch and somehow missed [2.4](https://github.com/tensorflow/tensorflow/blob/r2.4/tensorflow/python/keras/layers/pooling.py#L341) branch too. I guess keras.io site gets updated with latest stable TF version so it won't get fixed with 2.4 release unless changes get cherry picked (it's already Release Candidate 3). Maybe in 2.5?

**Comment by jvishnuvardhan** [comment]
@Molkree This is already updated in [Keras.io](https://keras.io/api/layers/pooling_layers/max_pooling2d/). The [`master`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/pooling.py#L333) branch has the updated code as mentioned above but I think this will be updated soon in TF website. Thanks!

**Comment by ymodak** [comment]
This is fixed in latest tf nightly. Thanks!
https://www.tensorflow.org/api_docs/python/tf/keras/layers/MaxPool2D?version=nightly
https://github.com/tensorflow/tensorflow/blob/d7fc63e468546c4bcbaaeabb1f3b3a09deb4e894/tensorflow/python/keras/layers/pooling.py#L341

**Comment by jvishnuvardhan** [comment]
Closing this issue as this was resolved already. Thanks!
